bcda/client/bluebutton.go

```diff
@@ -238,7 +245,7 @@ func GetDefaultParams() (params url.Values) {
 	return params
 }
 
-func HashHICN(toHash string) (hashedValue string) {
+func HashIdentifier(toHash string) (hashedValue string) {
 	blueButtonPepper := os.Getenv("BB_HASH_PEPPER")
```

Comment:
Need to confirm if these values will be different between `hicn-hash` and `mbi-hash`.

Reply:
received 01/06/2020:
"Received confirmation from BFD that the hashing algorithm will be the same for HICN and MBI. No need to change iterations, or pepper, etc"